README.md

```diff
@@ -81,10 +69,6 @@ Will setup container runtime on your host instance.
     ./main.sh up
     ```
 
-1. kubeconfig
```

Comment:
we should keep this.

Reply:
Can you explain why we need to keep this?

Reply:
we still have kind provider in this repo, so when people use kind to set up their cluster with this repo. he/she should get kubeconfig for kind.